controller/source/rocksdb_server/server.cpp

```diff
@@ -20,44 +27,42 @@ class session : public std::enable_shared_from_this<session> {
   session(tcp::socket socket, std::shared_ptr<rocksdb_proxy> rocksdb_proxy)
       : m_socket(std::move(socket))
       , m_rocksdb_proxy(std::move(rocksdb_proxy))
+      , m_strand(io_service)
   {
   }
 
   void start() {
-    handle_read();
+    auto self = shared_from_this();
+    m_strand.post([self]() {self->handle_read();});
```

Comment:
Based on the [documentation](https://www.boost.org/doc/libs/1_81_0/doc/html/boost_asio/reference/io_context__strand.html), the post function call from the strand object might be deprecated. Can you verify and possibly adapt to the new version, to avoid any issues?

Reply:
As agreed offline, this is no longer needed as we switch to creating a thread per connection and not using a pool.